src/Forms/GridField/GridFieldDetailForm_ItemRequest.php

```diff
@@ -271,73 +271,79 @@ public function ItemEditForm()
         return $form;
     }
 
+    /**
+     * @return CompositeField Returns the right aligned toolbar group field along with its FormAction's
+     */
+    private function getRightGroupField()
```

Comment:
Please, we prefer protected over private.

There's also some other code looking for `$rightGroup` which is redundant now that this code is separated.

PR incoming

Reply:
It'd be also cool to add some more details to the PR description - either a summary of what you've changed, and/or a reference issue for what this change is related to